packages/cli/src/commands/info.js

```diff
@@ -19,70 +21,112 @@ export default class InfoCommand extends Command {
     summary: 'list info',
     version: '1.0.0',
     args: [],
-    options: [
-      [ '--debug', 'include os versions in output' ],
-    ],
```

Comment:
If the linter permits it, having each option array on a separate line is consistent across commands.